style.js

```diff
@@ -23,12 +23,156 @@ const lessButton = document.querySelector('.show-less');
 function showB() {
   dispNone.style.display = 'block';
   moreButton.style.display = 'none';
+  lessButton.style.display = 'flex';
 }
 
 function hideB() {
   dispNone.style.display = 'none';
   moreButton.style.display = 'block';
+  lessButton.style.display = 'none';
 }
 
 moreButton.addEventListener('click', showB);
-lessButton.addEventListener('click', hideB);
\ No newline at end of file
+lessButton.addEventListener('click', hideB);
+
+const mainProgram = document.getElementById('mainProgram');
+
+const prog = [{
+  id: 1,
+  image: './images/lecture.png',
+  headingProgram: 'Lecture',
+  paragraphProgram: 'Listen to the speaker from various countries about sharing and opening',
+},
+{
+  id: 2,
+  image: './images/exhibition.png',
+  headingProgram: 'CC Exhibition',
+  paragraphProgram: 'Listen to the speaker from various countries about sharing and opening',
+},
+{
+  id: 3,
+  image: './images/forum.png',
+  headingProgram: 'Forum',
+  paragraphProgram: 'Listen to the speaker from various countries about sharing and opening',
+},
+{
+  id: 4,
+  image: './images/workshop.png',
+  headingProgram: 'Workshop',
+  paragraphProgram: 'Listen to the speaker from various countries about sharing and opening',
+},
+{
+  id: 5,
+  image: './images/brain.png',
+  headingProgram: 'CC Ignite',
+  paragraphProgram: 'Listen to the speaker from various countries about sharing and opening',
+},
+
+];
+
+const programCards = prog
+
+  .map(
+    (card) => `
+
+        
+        <div class="card">
+            <img src="${card.image}" alt="lecture">
+
+            <h3>${card.headingProgram}</h3>
+            <p>${card.paragraphProgram}</p>
+        </div>
+  
+
+`,
+  )
+  .join('');
+
+mainProgram.innerHTML += programCards;
+
+const featuredSpeakers = document.getElementById('featuredSpeakers');
+
+const speakers = [{
+  id: 1,
+  img: './images/Picture3.png',
+  headingThree: 'FirstName LastName',
+  headingFive: 'Bickerman Professor of Entreprenurial Legal Studies at Havard Law School',
+  speakerParagraph: 'Listen to the speaker from various countries about sharing and opening',
+},
+{
+  id: 2,
+  img: './images/Picture3.png',
+  headingThree: 'FirstName LastName',
+  headingFive: 'Bickerman Professor of Entreprenurial Legal Studies at Havard Law School',
+  speakerParagraph: 'Listen to the speaker from various countries about sharing and opening',
+},
```

Comment:
It's important that the list of speakers contain a speaker object for each that holds their specific data. Kindly change their info a bit so that they are distinct from each other 😬